gobblin-core/src/main/java/gobblin/source/extractor/filebased/FileBasedExtractor.java

```diff
@@ -192,4 +209,14 @@ public void close() {
       LOGGER.error("Could not successfully close file system helper due to error: " + e.getMessage(), e);
     }
   }
+
+  private void incrementBytesReadCounter() {
+    try {
+      counters.inc(CounterNames.FileBytesRead, fsHelper.getFileSize(currentFile));
+    } catch (FileBasedHelperException e) {
+      LOGGER.info("Unable to get file size. Will skip increment to bytes counter " + e.getMessage());
```

Comment:
Since it was an info I did not want to print the complete stacktrace. May be we can print it in a LOGGER.Warn but it may be an overkill for a missing counter. I can add a debug message with the stacktrace